.travis.yml

```diff
@@ -6,9 +6,10 @@ matrix:
       jdk: oraclejdk9
     - os: osx
 
-script: >-
-    ./config/travis/run-checks.sh &&
-    time travis_retry ./gradlew clean checkstyleMain checkstyleTest test functional
+script:
+  - ./config/travis/run-checks.sh
+  - cd ./frontend; npm run lint src/static/js/*js; cd ..
```

Comment:
why dont just make it frontend/src

Reply:
are you referring to the file path? 

Well, eslint is kind of a pesky fellow, on my side, it doesnt recursively lints through all the files, well, it claims it does, but it doesnt, and it says its some bug thingy.

All the JS files are in one place anyways, so i figured, why not?

Reply:
i mean, why do you need to cd if you can just append ./frontend in front of src?